spec/hbw/features/file_upload_spec.rb

```diff
@@ -72,5 +72,23 @@
       expect(preview_row).to have_no_text 'logo.svg'
       expect(preview_row).to have_no_css  "img[alt='logo.svg']"
     end
+
+    scenario 'should attach to second file_upload' do
+      attach_files('homsOrderDataUploadedFileAnother', files_to_attach)
+
+      preview_another = page.all('.files-preview-row')[0]
+      form_group = page.all('.form-group')[0]
+      file_input = form_group.find_field('homsOrderDataUploadedFileAnother', type: :file, visible: :hidden)
+
+      expect(file_input['name']).to  eq('homsOrderDataUploadedFileAnother')
+
+      expect(preview_another).to have_content 'logo.svg'
+      expect(preview_another).to have_content 'file.txt'
+      expect(preview_another).to have_content 'file_with...me.pdf'
```

Comment:
I think you should do something equivalent to your gif: attach files to both inputs in the same test.